panacea/query_client.go

```diff
@@ -510,7 +517,11 @@ func (q verifiedQueryClient) GetLastBlockHeight(ctx context.Context) (int64, err
 	return trustedBlock.Height, nil
 }
 
-func (q verifiedQueryClient) VerifyTrustedBlockInfo(height int64, blockHash []byte) error {
+func (q *verifiedQueryClient) GetCachedLastBlockHeight() int64 {
```

Comment:
It doesn't seem to be used - other functions seem to call `q.cachedLastBlockHeight` directly.

Reply:
OK. I fixed it (https://github.com/medibloc/panacea-oracle/pull/101/commits/b1bb69642e94515a52cf4643f820e8fb6debf736)